README.md

```diff
@@ -1,6 +1,56 @@
 # CWMSpy
 CWMS REST API for Data Retrieval
 
+## Development
```

Comment:
I recommend separating out into a `CONTRIBUTING.md` file similar to other HEC managed repositories:
https://github.com/USACE/cwms-data-api/blob/develop/CONTRIBUTING.md
https://github.com/opendcs/opendcs/blob/master/CONTRIBUTING.md

That way the README.md can be simplified to general repository/usage type information.

Reply:
Thanks, that's a much better place for it.

Reply:
I pushed up a change with a `CONTRIBUTING.md` file. It doesn't include any of the DCO stuff at the moment (I didn't feel comfortable adding legal info without checking first). I'll open an issue to add this in a future commit.